app/services/piv_cac/check_config.rb

```diff
@@ -0,0 +1,14 @@
+module PivCac
+  class CheckConfig
+    def self.call
+      return unless Rails.env.production?
+
+      url = URI.parse(Figaro.env.piv_cac_verify_token_url)
+      return unless url.scheme == 'https'
+
+      message = "PIV/CAC configured without SSL: #{Figaro.env.piv_cac_verify_token_url}"
+      Rails.logger.error { "#{message} - EXITING" }
```

Comment:
Can we also notify NewRelic? Might be easier to find than the logs

Reply:
If this gets raised, the app fails to start at all so it should be extremely obvious. Still useful to send to NewRelic?

Reply:
I think so? I find it easier to see errors in NR than digging through the startup logs for them

Reply:
Done!

Reply:
Thanks!